package/MDAnalysis/coordinates/XYZ.py

```diff
@@ -365,7 +365,7 @@ def _read_next_timestep(self, ts=None):
             f.readline()
             f.readline()
             for i in range(self.n_atoms):
-                self.ts._pos[i] = list(map(float, f.readline().split()[1:4]))
+                self.ts._pos[i] = np.float32(f.readline().split()[1:4])
```

Comment:
And weirdly, it's quicker just to do `self.ts._pos[i] = f.readline().split()[1:4]` rather than try and do the conversion yourself